### ISSUE TITLE
Support for request signing

### ISSUE THREAD

**Issue by JordanGottardoDocplanner** [seven-io/net-client#2]
Hi, I would like to use request signing as per the [official documentation](https://www.sms77.io/en/docs/gateway/http-api/signing-of-requests/). I am not able to find any of this functionality in this client. Is this not supported?

**Comment by matthiez** [comment]
Hi, unfortunately this is not implemented yet. We are working on a solution and will get back to you ASAP.

**Comment by matthiez** [comment]
Hi again,
you can now set the fourth client parameter as your signing secret and it will validate each request. Does that help?

**Comment by JordanGottardoDocplanner** [comment]
Yes, that would definitely help! Could you please release this new version as a nuget package?

A question: where do I need to retrieve this signing secret? The docs says "You can find the key in your login in the developer area in the API settings", but I find no mention of this in the dashboard. The only key I can find is the API key. Do I need to use this?

**Comment by matthiez** [comment]
Hi, we are working on publishing it, sorry for the delay. Having troubles signing into NuGet so waiting for Microsoft right now. You can find the secret by clicking on the cog in your developer dashboard. See image.
<img width="363" alt="image" src="https://user-images.githubusercontent.com/12965261/191754975-5a1d91f5-8663-40d1-8752-3fae4f6020e7.png">
